src/coreclr/debug/ee/controller.cpp

```diff
@@ -3879,6 +3887,73 @@ void DebuggerController::DispatchMethodEnter(void * pIP, FramePointer fp)
 
 }
 
+void DebuggerController::EnableMultiCastDelegate()
+{
+    CONTRACTL
+    {
+        NOTHROW;
+        GC_NOTRIGGER;
+    }
+    CONTRACTL_END;
+
+    ControllerLockHolder chController;
+    if (!m_multicastDelegateHelper)
+    {
+        LOG((LF_CORDB, LL_INFO1000000, "DC::EnableMultiCastDel, this=%p, previously disabled\n", this));
+        m_multicastDelegateHelper = true;
+    }
+    else
+    {
+        LOG((LF_CORDB, LL_INFO1000000, "DC::EnableMultiCastDel, this=%p, already set\n", this));
+    }
+    g_multicastDelegateTraceEnabled += 1;
```

Comment:
Does `g_multicastDelegateTraceEnabled` operate as a reference count as well? If so, I would rename it. If not, instead of `+= 1`, it should be `= 1`. and the disable should be `= 0`

Reply:
> It is important to have this be a reference count. 

Thanks for the confirmation. I would suggest calling it `g_multicastDelegateTraceUseCount`, `g_multicastDelegateTraceActiveCount`, or something similar.